heliopy/data/helios.py

```diff
@@ -957,6 +958,19 @@ def merged(probe, starttime, endtime, try_download=True):
                        'helios{}/merged/he{}_40sec'.format(probe, probe))
     fnames = []
     dirs = []
+    units = OrderedDict([('rh', u.AU), ('esh', u.deg),
+                        ('clong', u.deg), ('clat', u.deg), ('HGIlong', u.deg),
+                        ('br', u.nT), ('bt', u.nT), ('bn', u.nT),
+                        ('vp1r', u.km / u.s),
+                        ('vp1t', u.km / u.s), ('vp1n', u.km / u.s),
+                        ('crot', u.dimensionless_unscaled), ('np1', u.cm**-3),
+                        ('vp1', u.km / u.s), ('Tp1', u.K), ('vaz', u.deg),
+                        ('vel', u.deg), ('Bx', u.nT), ('By', u.nT),
+                        ('Bz', u.nT), ('sBx', u.dimensionless_unscaled),
```

Comment:
`sBx` etc. should have units of `u.nT`, since they're the standard deviation of a quantity that also has units `u.nT`.

Reply:
Oh yep!